lib/main_publiccloud.pm

```diff
@@ -40,6 +44,9 @@ sub load_maintenance_publiccloud_tests {
     } elsif (get_var('PUBLIC_CLOUD_NEW_INSTANCE_TYPE')) {
         loadtest("publiccloud/bsc_1205002", run_args => $args);
     } else {
+        if (get_var('PUBLIC_CLOUD_REGISTRATION_TESTS')) {
+            loadtest("publiccloud/check_registercloudguest", run_args => $args);
+        }
         loadtest "publiccloud/ssh_interactive_start", run_args => $args;
```

Comment:
does it make sense to run `ssh_interactive_start`/`ssh_interactive_end` in this case ? AFAIK it meant to be "preparation before test" and if you actually run test BEFORE what the point to run them ?

Reply:
I would or move this "out" of this "else" ( below `elsif (get_var('PUBLIC_CLOUD_NEW_INSTANCE_TYPE')) {` ) or after `ssh_interactive_start` ( below `elsif (check_var('PUBLIC_CLOUD_NVIDIA', 1)) {` )

Reply:
It cannot be after `ssh_interactive_start` as there is the console already 'shifted' to the VM.
The `ssh_interactive_start` is there to prepare the environment for `instance_overview`.

Now thinking about it we might replace the `instance_overview` with `smoketest` tho :smile:

Reply:
Done.